### PR TITLE
Close issue #637 with tests

### PR DESCRIPTION
**Pull request by Hosch250**
Completely redid UI.  Did not remove duplicates from config file.

### CODE REVIEW

**Comment by ckuhn203** [comment]
http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Comment by ckuhn203** [comment]
Don't feel bad about the squash... I couldn't get it to work either. I'll go ahead and review the hard way. =;)-

File: RetailCoder.VBE/Settings/ConfigurationLoader.cs [hunk 1]
```diff
@@ -113,7 +114,7 @@ public Configuration GetDefaultConfiguration()
         public ToDoMarker[] GetDefaultTodoMarkers()
         {
             var note = new ToDoMarker(RubberduckUI.ToDoMarkerNote, TodoPriority.Low);
-            var todo = new ToDoMarker(RubberduckUI.ToDoMarkerToDo, TodoPriority.Normal);
+            var todo = new ToDoMarker(RubberduckUI.ToDoMarkerToDo, TodoPriority.Medium);
```
Comment (ckuhn203) [inline RetailCoder.VBE/Settings/ConfigurationLoader.cs]: .....This is a breaking change. I don't know about this. I know we said we'd stuff all the breaking changes for the config into this release though, so... okay. We really need to treat the serialized classes like we would treat COM interfaces though. It's always okay to add to the interface, but never to change or remove from it.

**Comment by ckuhn203** [comment]
The new form looks and works great. Since we're breaking the config for localization, I'm okay with the priority level break. Needs to be well documented though. I'll open an issue.

File: RetailCoder.VBE/Settings/ConfigurationLoader.cs (hunk 1 above)
Comment (Hosch250) [inline RetailCoder.VBE/Settings/ConfigurationLoader.cs]: I thought I might as well change that since we were displaying "Medium" anyway.

**Pull request merged**